src/views/testUtils.js

```diff
@@ -17,7 +17,7 @@ export const visit = path => {
     // From the docs: Note that, even though goto normally waits
     // for all the resources on a page to load, a timeout exception
     // is only raised if the DOM itself has not yet loaded.
-    gotoTimeout: 4000
+    gotoTimeout: process.env.TEST_TIMEOUT || 4000
```

Comment:
This timeout is not the one causing the tests to fail. If anything, lowering it might actually make them pass.

This is what happens on a single test:
1. Start an electron instance, i.e. a browser. (slow)
```js
const nightmare = Nightmare(config)
```
2. Tell the browser to navigate to a page. (also slow)
```js
nightmare.goto(location)
```
This resolves successfully when the whole page finishes loading. That includes images. If the gotoTimeout is reached then it checks if at least the DOM is ready, in that case it resolves successfully anyway, that is the reason why lowering it might help. Otherwise it throws a timeout exception. This is not what is happening in the screenshots from #32.

3. Then it runs the rest of the test code. (quite fast)

All of this is constrained within jest's timeout for the tests, which is 5s by default. This is the timeout that is being triggered.

That timeout can be configured like this ([as per the docs](https://facebook.github.io/jest/docs/troubleshooting.html)):
```js
jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000 // 10 seconds
// yes, it's jasmine instead jest. idk why
```
Maybe this if the linter complains:
```js
global['jasmine'].DEFAULT_TIMEOUT_INTERVAL = 10000
```
There is also [jest.setTimeout()](http://facebook.github.io/jest/docs/en/jest-object.html#jestsettimeouttimeout). Maybe if the other does not work try with this (?)

As for where to put it, since all the long running tests are the ones that import testUtils, I think it should live in that file at the top level.